samza-core/src/main/java/org/apache/samza/container/grouper/task/TaskAssignmentManager.java

```diff
@@ -103,21 +106,53 @@ public TaskAssignmentManager(MetadataStore taskContainerMappingMetadataStore, Me
    * @param taskMode the mode of the task
    */
   public void writeTaskContainerMapping(String taskName, String containerId, TaskMode taskMode) {
-    String existingContainerId = taskNameToContainerId.get(taskName);
-    if (existingContainerId != null && !existingContainerId.equals(containerId)) {
-      LOG.info("Task \"{}\" in mode {} moved from container {} to container {}", new Object[]{taskName, taskMode, existingContainerId, containerId});
-    } else {
-      LOG.debug("Task \"{}\" in mode {} assigned to container {}", taskName, taskMode, containerId);
-    }
+    writeTaskContainerMappings(ImmutableMap.of(containerId, ImmutableMap.of(taskName, taskMode)));
```

Comment:
would it be good to replace `wirteTaskContainerMapping` with the new `wirteTaskContainerMappings` altogether?
there seems to be only one usage of this method and that has been replaced to use the new one in JobModelManager. 
is there a reason to keep this old method?